src/main/java/com/royalrangers/controller/achievement/UserRewardController.java

```diff
@@ -0,0 +1,65 @@
+package com.royalrangers.controller.achievement;
+
+import com.royalrangers.dto.ResponseResult;
+import com.royalrangers.service.achievement.UserRewardService;
+import com.royalrangers.utils.ResponseBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Map;
+
+@RestController
+@RequestMapping("/achievements/userReward")
+public class UserRewardController {
+    @Autowired
+    private UserRewardService userRewardService;
+
+    @GetMapping
+    public ResponseResult getAllRewardsForUser(){
+        try{
+            return ResponseBuilder.success(userRewardService.getAllRewardForUser());
+        } catch (Exception ex) {
+            return ResponseBuilder.fail("Failed get all UserReward");
+        }
+    }
+
+    @PostMapping
+    public ResponseResult addUserReward(@RequestBody Map<String, Object> params){
```

Comment:
We all agreed not to use such a statements:
```
@RequestBody Map<String, Object> params
```
We're crating appropriate DTO for that